src/engine/framework/System.cpp

```diff
@@ -751,6 +778,78 @@ ALIGN_STACK_FOR_MINGW int main(int argc, char** argv)
 		Sys::Error("Unhandled exception of unknown type");
 	}
 
+	// Must be done after Sys::Init() to read cvars from command line.
```

Comment:
If you want to do it earlier, you can either
(a) use `EarlyCvar`, or
(b) use `Cvar::INIT` and then do it any time after the call to SetCvarsWithInitFlag.

Putting this stuff into a separate function would be nice BTW.

Reply:
I've put this stuff in a dedicated function and called it right after `SetCvarsWithInitFlag()`.